lib/v1/config.js

```diff
@@ -13,8 +18,5 @@ module.exports = (teamId) => ({
       version: 1,
     },
   },
-  http: {
-    keysUrl: KEYS_URL,
-    metricsUrl: METRICS_URL,
-  }
+  http: configUrls
```

Comment:
configUrls can be anything here including undefined which would break the SDK. We need to add a check to ensure that it provides the expected urls otherwise we should fallback to the defaults